health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/securityPopUp.js

```
@@ -0,0 +1,93 @@
+      ...prev,
+      [id]: value,
+    }));
+  };
+
+  return (
+    <PopUp
+      onClose={onClose}
+      heading={t(`HCM_MICROPLAN_VILLAGE_SECURITY_LEVEL_LABEL`)}
+      onOverlayClick={onClose}
+      children={[
+        <Card type="secondary">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)